<?xml version="1.0" encoding="utf-8" standalone="no"?>
<test-results xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="nunit_schema_2.5.xsd" name="Pester" total="66" errors="0" failures="52" not-run="0" inconclusive="0" ignored="0" skipped="2" invalid="0" date="2021-09-14" time="14:53:46">
  <environment cwd="C:\Users\Laurent.Lienhard\SynologyDrive\01-DEV\POSHTools\tests" os-version="10.0.19043" user-domain="NETINTRA" platform="Microsoft Windows 10 Entreprise|C:\Windows|\Device\Harddisk0\Partition3" nunit-version="2.5.8.0" user="Laurent.Lienhard" clr-version="Unknown" machine-name="PORT9999" />
  <culture-info current-culture="fr-FR" current-uiculture="fr-FR" />
  <test-suite type="TestFixture" name="Pester" executed="True" result="Failure" success="False" time="1.5278" asserts="0" description="Pester">
    <results>
      <test-suite type="TestFixture" name="C:\Users\Laurent.Lienhard\SynologyDrive\01-DEV\POSHTools\tests\Help.tests.ps1" executed="True" result="Failure" success="False" time="0.794" asserts="0" description="C:\Users\Laurent.Lienhard\SynologyDrive\01-DEV\POSHTools\tests\Help.tests.ps1">
        <results>
          <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0.1363" asserts="0" description="Test help for &lt;_.Name&gt;">
            <results>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-case description="Help is not auto-generated" name="Test help for &lt;_.Name&gt;.Help is not auto-generated" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example code" name="Test help for &lt;_.Name&gt;.Has example code" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example help" name="Test help for &lt;_.Name&gt;.Has example help" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0.1697" asserts="0" description="Test help for &lt;_.Name&gt;">
            <results>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-case description="Help is not auto-generated" name="Test help for &lt;_.Name&gt;.Help is not auto-generated" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example code" name="Test help for &lt;_.Name&gt;.Has example code" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example help" name="Test help for &lt;_.Name&gt;.Has example help" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0.2092" asserts="0" description="Test help for &lt;_.Name&gt;">
            <results>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-case description="Help is not auto-generated" name="Test help for &lt;_.Name&gt;.Help is not auto-generated" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example code" name="Test help for &lt;_.Name&gt;.Has example code" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example help" name="Test help for &lt;_.Name&gt;.Has example help" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0.2447" asserts="0" description="Test help for &lt;_.Name&gt;">
            <results>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-case description="Help is not auto-generated" name="Test help for &lt;_.Name&gt;.Help is not auto-generated" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example code" name="Test help for &lt;_.Name&gt;.Has example code" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example help" name="Test help for &lt;_.Name&gt;.Has example help" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
            </results>
          </test-suite>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="C:\Users\Laurent.Lienhard\SynologyDrive\01-DEV\POSHTools\tests\Manifest.tests.ps1" executed="True" result="Ignored" success="True" time="0.5229" asserts="0" description="C:\Users\Laurent.Lienhard\SynologyDrive\01-DEV\POSHTools\tests\Manifest.tests.ps1">
        <results>
          <test-suite type="TestFixture" name="Module manifest" executed="True" result="Success" success="True" time="0.328" asserts="0" description="Module manifest">
            <results>
              <test-suite type="TestFixture" name="Module manifest.Validation" executed="True" result="Success" success="True" time="0.3175" asserts="0" description="Module manifest.Validation">
                <results>
                  <test-case description="Has a valid manifest" name="Module manifest.Validation.Has a valid manifest" time="0.132" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid name in the manifest" name="Module manifest.Validation.Has a valid name in the manifest" time="0.0305" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid root module" name="Module manifest.Validation.Has a valid root module" time="0.0098" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid version in the manifest" name="Module manifest.Validation.Has a valid version in the manifest" time="0.0126" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid description" name="Module manifest.Validation.Has a valid description" time="0.0242" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid author" name="Module manifest.Validation.Has a valid author" time="0.0112" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid guid" name="Module manifest.Validation.Has a valid guid" time="0.0297" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid copyright" name="Module manifest.Validation.Has a valid copyright" time="0.018" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid version in the changelog" name="Module manifest.Validation.Has a valid version in the changelog" time="0.0226" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Changelog and manifest versions are the same" name="Module manifest.Validation.Changelog and manifest versions are the same" time="0.0171" asserts="0" success="True" result="Success" executed="True" />
                </results>
              </test-suite>
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="Git tagging" executed="True" result="Ignored" success="True" time="0.3445" asserts="0" description="Git tagging">
            <results>
              <test-case description="Is tagged with a valid version" name="Git tagging.Is tagged with a valid version" time="0.0089" asserts="0" success="False" result="Ignored" executed="False" />
              <test-case description="Matches manifest version" name="Git tagging.Matches manifest version" time="0.0035" asserts="0" success="False" result="Ignored" executed="False" />
            </results>
          </test-suite>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="C:\Users\Laurent.Lienhard\SynologyDrive\01-DEV\POSHTools\tests\Meta.tests.ps1" executed="True" result="Success" success="True" time="0.2109" asserts="0" description="C:\Users\Laurent.Lienhard\SynologyDrive\01-DEV\POSHTools\tests\Meta.tests.ps1">
        <results>
          <test-suite type="TestFixture" name="Text files formatting" executed="True" result="Success" success="True" time="0.0513" asserts="0" description="Text files formatting">
            <results>
              <test-suite type="TestFixture" name="Text files formatting.File encoding" executed="True" result="Success" success="True" time="0.0155" asserts="0" description="Text files formatting.File encoding">
                <results>
                  <test-case description="No text file uses Unicode/UTF-16 encoding" name="Text files formatting.File encoding.No text file uses Unicode/UTF-16 encoding" time="0.0129" asserts="0" success="True" result="Success" executed="True" />
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Text files formatting.Indentations" executed="True" result="Success" success="True" time="0.0485" asserts="0" description="Text files formatting.Indentations">
                <results>
                  <test-case description="No text file use tabs for indentations" name="Text files formatting.Indentations.No text file use tabs for indentations" time="0.0275" asserts="0" success="True" result="Success" executed="True" />
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>